### PR TITLE
Add logos block to content page

### PR DESCRIPTION
**Pull request by vossisboss**
We need some small adjustments to the Logos block and to the blocks available on the Content page to create a suitable MVP design for the "Who Uses Wagtail" page. 

This PR makes the Logos block available on the Content page type and provides an optional heading that can be used to organise the logos.

Ben E and Ian B from Torchbox want to review the design a bit before we proceed with merging, so I'm submitting this PR as a draft for now.

### CODE REVIEW

File: wagtailio/core/migrations/0056_auto_20221114_2049.py [hunk 1]
```diff
@@ -1,30 +1,1589 @@
 # Generated by Django 3.2.16 on 2022-11-14 20:49
```
Comment (zerolab) [inline wagtailio/core/migrations/0056_auto_20221114_2049.py:1]: suggestion: rename this migration. we try to steer away from auto-named migrations

**Comment by zerolab** [comment]
> Ben E and Ian B from Torchbox want to review the design a bit before we proceed with merging, so I'm submitting this PR as a draft for now.

This sounds like a perfect opportunity to get this onto `staging`. Don't have any qualms with the BE changes

**Comment by vossisboss** [comment]
@zerolab I sent Ben and Ian a short walkthrough of the design from my local machine. They approved it, so I think we can proceed to production. But I'm okay with testing on staging first if that's your preference.

File: wagtailio/static/sass/components/_logos.scss [hunk 2]
```diff
@@ -4,11 +4,11 @@
     position: relative;
 
     &__heading {
-        margin-bottom: 20px;
+        margin-bottom: 80px;
```
Comment (laymonage) [inline wagtailio/static/sass/components/_logos.scss:7]: We might want to reduce this, and maybe increase the margin below the logos instead? Feels a bit weird that the bottom margin of the heading is larger (80px) than the bottom margin between the logos and the next heading (50px), which is a separate group.

image


image

**Review by laymonage — COMMENTED** [review]
Changes to the blocks look good to me, just a minor issue with the spacing

File: wagtailio/static/sass/components/_logos.scss (hunk 2 above)
Comment (vossisboss) [reply wagtailio/static/sass/components/_logos.scss]: Submitted a commit that sets the margin to `60px` for all views. Please let me know if there's anything else I should fix.

**Review by laymonage — APPROVED** [review]

**Pull request merged**